x-pack/test/functional/services/ml/job_table.ts

```diff
@@ -373,6 +373,19 @@ export function MachineLearningJobTableProvider(
       await testSubjects.existOrFail('~mlPageJobWizard');
     }
 
+    public async clickCloneJobActionWhenNoDataViewExists(jobId: string) {
+      await this.ensureJobActionsMenuOpen(jobId);
+      await testSubjects.click('mlActionButtonCloneJob');
+      await this.assertNoDataViewForCloneJobWarningToastExist();
+    }
+
+    public async assertNoDataViewForCloneJobWarningToastExist() {
+      await retry.tryForTime(5000, async () => {
+        const toast = await testSubjects.find('mlCloneJobNoDataViewExistsWarningToast');
+        expect(toast).not.to.be(undefined);
+      });
```

Comment:
This is working fine. However, there's a shortcut for this (no need to change though):
```suggestion
      await testSubjects.existOrFail('mlCloneJobNoDataViewExistsWarningToast', { timeout: 5000 });
```

Reply:
Switched to the concise version in https://github.com/elastic/kibana/pull/117993/commits/b67a413b1d958a3c05428f18fa6218320ebd8a11.